ledger-api/grpc-definitions/com/daml/ledger/api/v1/expected_ledger_feature_descriptors.yaml

```diff
@@ -0,0 +1,58 @@
+# Example feature descriptor for our existing ledgers.
+# Written as pseudo YAML using its mapping to JSON and then to `FeatureDescriptor` proto message
+
+sandbox-classic:
+  invocation_matrix:
+    # TODO
+  ledger:
+    ordered_contract_ids: 
+      # set only if contract-id seeding is not weak
+    monotonic_record_time:
+  participant:
+    party_allocation:
+      implicit_party_allocation: True 
+    participant_pruning:
+      supports_pruning_all_divulged_events: False
+    ledger_config_management:
+  experimental:
+    rank_based_command_deduplication:
+      completions_include_submission_ids: True
+
+sandbox-next:
+  invocation_matrix:
+    - "with and without --enable-append-only-schema"
+    - "wall-clock-time vs. static time"
+    - "with and without --implicit-party-allocation=false"
+    - "h2 vs. postgres"
+    - "lf versions to test: default, latest, preview"
+  ledger:
+    ordered_contract_ids: 
+      # set only if contract-id seeding is not weak
+    monotonic_record_time:
+  participant:
+    party_allocation:
+      implicit_party_allocation: #set to the value given on command line 
+    participant_pruning:
+      supports_pruning_all_divulged_events: True
+    ledger_config_management:
+  experimental:
+    rank_based_command_deduplication:
+      completions_include_submission_ids: True
+
+
+canton:
+  invocation_matrix:
+    - "uck vs. non-uck"
+    # TODO: there are likely more.
```

Comment:
For example:
* Postgres vs. H2 vs. Oracle
* enterprise vs. community (although this influences the available features, so maybe better to have to separate versions)
* HA on/off